hashes/xxhash.py

```diff
@@ -0,0 +1,112 @@
+# Author: Your Name
+# Author email: your.email@example.com
+# Coding date: Month Year
+# Black: True
+
+"""
+    * This code implements the xxHash algorithm:
+        https://github.com/Cyan4973/xxHash
+
+    * xxHash is an extremely fast non-cryptographic hash algorithm that can be used
+      to hash data for various applications like hash tables, checksums, and more.
+
+    * The implemented code consists of:
+        * A function for calculating the xxHash hash value for given data.
+
+    * How to use:
+        You can use the `xxhash` function to calculate the hash value for your data.
+
+    * Example:
+        data = b'Hello, World!'
+        seed = 0
+        hash_value = xxhash(data, seed)
+        print(f'Hash value: {hash_value}')
+
+"""
+
+# Constants
+PRIME32_1 = 0x9E3779B1
+PRIME32_2 = 0x85EBCA77
+PRIME32_3 = 0xC2B2AE3D
+PRIME32_4 = 0x27D4EB2F
+PRIME32_5 = 0x165667B1
+
+
+def xxhash(data, seed=0):
+    """
+    Calculate the xxHash hash value for the given data.
+
+    :param data: Data to be hashed (bytes)
+    :param seed: Seed value for the hash (integer)
+    :return: xxHash hash value (integer)
```

Comment:
DRY: Don't repeat yourself
```suggestion
    :param data: Data to be hashed
    :param seed: Seed value for the hash
    :return: xxHash hash value
```